docs/tutorial/basic-dl-query.md

```diff
@@ -0,0 +1,46 @@
+# DL query
+
+This tutorial is based off https://ontology101tutorial.readthedocs.io/en/latest/DL_QueryTab.html
+Created by: Melissa Haendel, Chris Mungall, David Osumi-Sutherland, Matt Yoder, Carlo Torniai, and Simon Jupp
+
+## DL query tab
+
+The DL query tab shown below provides an interface for querying and searching an ontology. The ontology must be classified by a reasoner before it can be queried in the DL query tab.
+
+For this tutorial, we will be using cc.owl which can be found [here](https://github.com/OHSUBD2K/BDK14-Ontologies-101/tree/master/BDK14_exercises/basic-dl-query).
+
+Go to the “basic-dl-query” folder and open “cc.owl”. Run the reasoner. Navigate to the DL Query tab.
```

Comment:
For folders and filenames in general (not necessary, just as a convention) I kinda prefer `tick-syntax` over "quotes".. Up to you though.

Reply:
honestly didnt think too much about it, just copied and pasted as is lol